### ISSUE TITLE
Screensaver perpetually downloads video

### ISSUE THREAD

**Issue by JohnWilliston** [OrangeJedi/Aerial#147]
**Describe the bug**
The screensaver perpetually downloads video and seemingly doesn't cache it. I've tried uninstalling/reinstalling, changing the cache location, etc., but it just keeps downloading. My PC was averaging 90 GB a day downloads until I realized what was happening.

**To Reproduce**
1. Install the screensaver.
2. Let it run.

**Expected behavior**
I expected it to cache the videos after downloading, so I'd only have to do so once for each file.

**Screenshots**

**Aerial Version**
The latest (1.2.0 downloaded 5/25/2023.

**Additional context**

**Comment by OrangeJedi** [comment]
Did you enable caching? You can do this from the `Video Cache` tab pictured here:
![image](https://github.com/OrangeJedi/Aerial/assets/25063853/26fca915-36e4-4312-b90f-f3051de35010)

Aerial does not cache videos by default because of the large amount of space (~20-30GB) they can take up. It sounds like some welcome screen with set up options would be a good idea in the future.

**Comment by JohnWilliston** [comment]
I thought so, but now I can't get the settings dialog to come up. I see the process is running, but there's no icon in my system tray, launching the application doesn't seem to do anything, and I can't select it as a screensaver in the screen savers dialog anymore. I don't know what happened. Perhaps it's time to uninstall/reinstall?

UPDATE: I figured out why the screensaver was no longer working. After it seemed like it was failing to cache videos in the folder I told it to, I deleted that folder. If the folder listed in the config.json file doesn't exist, the app will never show its UI on the screen at all. Only by hand-editing that config.json file could I again interact with the software. Now I've ticked the box you mentioned (it wasn't checked before) and will try to figure out how to re-open this issue if it still fails to cache videos.
